### PR TITLE
User Profile edit added

### LINKED ISSUE #27: Edit User Profile
Write a post API through which user can edit these fields `github, Linkedin, mastercv, resume1, resume2`
All these should be valid links (write relevant regex check)

### PR DESCRIPTION
**Pull request by parinayc20**
Some unnecessary changes in some files are due to the use of black
closes #27

### CODE REVIEW

File: user/views.py [hunk 1]
```diff
@@ -114,3 +111,43 @@ def post(self, request):
...
+                        user.resume1 = request.data["resume1"]
+                    else:
+                        return Response(response1, status=status.HTTP_400_BAD_REQUEST)
+                if "resume2" in request.data:
+                    if linkValidator(request.data["resume2"]):
+                        user.resume2 = request.data["resume2"]
+                    else:
+                        return Response(response1, status=status.HTTP_400_BAD_REQUEST)
+                user.save()
+                response2 = {"message": "User registered successfully"}
```
Comment (namangup) [inline user/views.py]: Change message string. Profile edited or something like that.

File: user/utils.py [hunk 2]
```diff
@@ -13,6 +15,14 @@ def CHECK_PASSWORD(password, hash):
...
 
 
+def linkValidator(link):
+    regex = "^[https://]"
+    if not re.search(regex, link):
+        link = "https://" + link
+    is_valid = validators.url(link)
+    return is_valid
+
+
```
Comment (namangup) [inline user/utils.py]: You could maybe include a parameter `domain` which checks if the domain is correct.
`github.com` `linkedin.com` `google.com`
Frontend would also have these checks but it's better to have them here too for completeness

**Comment by namangup** [comment]
LGTM. Resolve conflicts and make requested changes.

**Comment by parinayc20** [comment]
Made all the required changes.

**Pull request merged**